system/jhtree/jhtree.cpp

```diff
@@ -2717,9 +2717,12 @@ class CKeyMerger : public CKeyLevelManager
                 stats.noteSkips(lskips, lnullSkips);
                 if (found)
                 {
-                    IKeyCursor *mergeCursor = LINK(keyCursor);
+                    IKeyCursor *mergeCursor = keyCursor;
```

Comment:
This does not feel right. mergeCursor is supposed to use keyCursor or a fixed version of it...

Isn't the fix to say Linked<mergeCursor> = keyCursor... mergeCurser.setown()...

Reply:
Pushed a second version